auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -83,7 +105,88 @@
       </template>
 
       <!-- Actions -->
-      <template v-slot:item-slot-Actions="{ item, index }">
+      <template v-if="enableNewActionsMenu()" v-slot:item-slot-Actions="{ item, index }">
+        <div class="new-actions mx-auto" :id="`action-menu-${index}`">
+          <span class="open-action">
+            <v-tooltip top content-class="top-tooltip" :disabled="disableTooltip(item)">
+              <template v-slot:activator="{on}">
+                <v-btn
+                  small
+                  color="primary"
+                  min-width="5rem"
+                  min-height="2rem"
+                  class="open-action-btn"
+                  v-on="on"
+                  @click="action(item)"
+                >
+                  {{ getPrimaryAction(item) }}
+                <v-icon class='external-icon pl-1' v-if="isOpenExternal(item)" small>mdi-open-in-new</v-icon>
+              </v-btn>
+              </template>
+              <span>Go to {{ getTooltipTargetDescription(item) }} to access this business</span>
```

Comment:
Does this get displayed for all primary actions?

Reply:
No, it only shows for those with externally open icons, except for `Download Form`.